Add clarification on how stolen beatmaps should be reported to `Help_centre/Beatmapping` (#9962)
ppy/osu-wiki: wiki/Help_centre/Beatmapping/en.md

If it isn't, manually change it for every difficulty in your beatmap, reload the beatmap, and you should be able to submit it.

If you are still having issues or want more information on this, have a read through the [BSS issues page](/wiki/Guides/BSS_issues).

## I am an osu!supporter, but I don't have 20 pending slots! {id=upload-slots}

**The number of slots available depends on the number of ranked maps you have. For details, see [osu!supporter § Increased limits](/wiki/osu!supporter#increased-limits).**

## My beatmap/content has been re-uploaded by another user without my consent, and I want the infringing copy to be taken down! {id=report-content-dmca}

> **Please file a DMCA claim to [copyright@ppy.sh](mailto:copyright@ppy.sh) as per the [osu! copyright policy](https://osu.ppy.sh/legal/en/Copyright).**

Reviewer: ```suggestion
**Please file a DMCA claim to [copyright@ppy.sh](mailto:copyright@ppy.sh) as per the [osu! copyright policy](/Legal/Copyright).**
```

> Stolen beatmaps are treated as a form of copyright infringement, and as such, their removal is subject to legal procedures. 

Reviewer 2: ```suggestion
Any kind of content uploaded without permission can be handled as copyright infringement if the original owner so wishes. Requesting removal requires a legal procedure in order to ensure fair proceedings.
```

> Please do **not** file your report using either [the in-game `!report` command](/wiki/Reporting_bad_behaviour#in-game-chat) or [the osu! website](/wiki/Reporting_bad_behaviour#website), as members of the [NAT](/wiki/People/Nomination_Assessment_Team) and [GMT](/wiki/People/Global_Moderation_Team) who manage these reports are not equipped to handle legal matters.

Reviewer 2: ```suggestion
Please do **not** file a report using either [the in-game `!report` command](/wiki/Reporting_bad_behaviour#in-game-chat) or [the osu! website](/wiki/Reporting_bad_behaviour#website), as members of the [NAT](/wiki/People/Nomination_Assessment_Team) and [GMT](/wiki/People/Global_Moderation_Team) who manage these reports are not equipped to handle legal matters.
```

**The number of slots available depends on the number of ranked maps you have. For details, see [osu!supporter § Increased limits](/wiki/osu!supporter#increased-limits).**

> ## My beatmap has been stolen or re-uploaded by another user without my consent, and I want the infringing copy to be taken down! {id=report-stolen-beatmap}

Reviewer 3: From a non-mapper perspective, is it worth specifying what counts as a beatmap being stolen, or is that implicitly well known?

Author: I don't think it's worth it tho, since most of the time what constitutes as "stealing" can be boiled down to common sense

e.g. people copying that one dorchadas slider to be used in a different map with a different context wouldn't be considered by many as "stealing"
but sth like [this](http://www.youtube.com/watch?v=xURjkQ6pjQQ) where a mapper is trying to pass off another mapper's creation as their own would definitely fall into "stealing"

Either way in the end on borderline cases the ones that will decide whether something is considered as a valid "beatmap stealing" are the legal team themselves so yeah

Reviewer 2: Please just remove the "stealing" term, it's stupid. The concept of "stealing" publicly available data is stupid.

```suggestion
## My beatmap/content has been re-uploaded by another user without my consent, and I want the infringing copy to be taken down! {id=report-content-dmca}
```
